flocker/node/agents/cinder.py

```diff
@@ -564,18 +568,27 @@ def detach_volume(self, blockdevice_id):
         )
 
     def destroy_volume(self, blockdevice_id):
+        """
+        Detach Cinder volume identified by blockdevice_id.
+
+        :raises TimeoutException: If the volume is not deleted
+            within the expected time
```

Comment:
Anything more we can expect in this case?  Is the volume going to be on its way to being deleted?  Or if this exception is raised then the volume won't be deleted unless further action is taken?